test/integration/dra/cluster/README.md

```diff
@@ -0,0 +1,43 @@
+This directory contains a testsuite with automatic upgrade/downgrade tests for
+DRA. Conceptually this is like an integration test, in the sense that it
+starts/stops cluster components and runs tests against them.
+
+The difference is that it starts Kubernetes components by running the actual
+binaries, relying on local-up-cluster.sh for the logic and configuration
+steps. Because local-up-cluster.sh needs additional permissions and
+preparations on the host, the test cannot run in "make test-integration" and
+just skips itself there.
```

Comment:
why not put it outside of integration folder then?

Reply:
It being under `test/integration/dra` has the practical advantage that all DRA integration tests, including this one, can be run with one command when setting up the environment accordingly.

But more importantly, I couldn't think of any other directory that would fit better. `test/e2e` is worse. A new `test/dra`? `test/other-integration/dra`? It would need a better name. Either way, we then have to add exceptions for not running the test in `make test` or keeping the self-skip, so it wouldn't be substantially different from where it is now.

Any suggestion?

Reply:
It seems you have done the exercise already, maybe @BenTheElder has some other idea, I do not know if e2e_dra can be an option, I can see we have e2e_kubeadm and e2e_node already

Reply:
We could do `test/e2e_dra`. Setting it up would imply modifying `hack/make-rules/test.sh`, but that's a one-time thing. No strong opinion from me.

Reply:
The boundary between E2E and integration is indeed fluid. It's about testing end-to-end flows (E2E) with a combination of several different components (integration).

I can imagine that this may become useful also in other areas, but for now I prefer to focus on DRA and explore a bit more how to write such tests before opening it up for others.